### DISCUSSION TITLE
More Details on Yarn 3

### DISCUSSION THREAD

**Discussion by wdfinch** [yarnpkg/berry#2802] [Q&A] [answered]
Hi all, I noticed that there are some RCs available for 3.0.0. I can't seem to find any information about 3.0.0 such as an RFC or release notes. Is there anywhere I can get more information on the next major version?

**Comment by arcanis** [comment, marked as answer]
Changelog at the root of the repo:
https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md

Umbrella issue pinned at the top of the issue page:
https://github.com/yarnpkg/berry/issues/1406

**Reply by wdfinch** [reply]
Thanks!
